packages/runtime/container-runtime/src/test/documentSchema.spec.ts

```diff
@@ -488,8 +495,8 @@ describe("Runtime", () => {
 			}, // onSchemaChange
 		);
 
-		// setting is not on yet
-		assert(controller3.sessionSchema.runtime.idCompressorMode === undefined);
+		// Attempted "off" -> "on" will transition to "delayed".
+		assert(controller3.sessionSchema.runtime.idCompressorMode === "delayed");
```

Comment:
If the existing container that is "off", but we know our desired state is "on", we coerce the 'idCompressorMode' to "delayed" so we're prepared to handle the migration operation.

This handles the case where a document was never loaded during the "off" -> "delayed" -> "on" transition: